apispec/ext/marshmallow/swagger.py

```diff
@@ -250,7 +250,7 @@ def field2property(field, spec=None, use_refs=True, dump=True, name=None):
         elif spec:
             ret.update(resolve_schema_dict(spec, field.schema))
```

Comment:
Yes, turns out it should be passed here as well; done in https://github.com/marshmallow-code/apispec/commit/cbe3f33b1f89dab2124d41d663e7633c8d6c8617